### PR TITLE
Us 18

### PR DESCRIPTION
**Pull request by smbende1**
Significant changes with the addition of a new GUI dialog and the objects to support it. At this time, the summary plan allows for creation only when one does not already exist. If one does exist, it goes into edit-only mode. When the project is complete, the actuals are retrieved from tasks. This currently applies only to the plan LOC field.

### CODE REVIEW

File: src/net/sf/memoranda/ui/PlanSummaryDialog.java [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+        	newprj = prjnew;
+        	psProject = passedPrj;
+            jbInit(prjName);
+            pack();
+        }
+        catch (Exception ex) {
+            new ExceptionDialog(ex);
+        }
+	}
+
```
Comment (krcarey1) [inline src/net/sf/memoranda/ui/PlanSummaryDialog.java:124]: formatting in the constructor

File: src/net/sf/memoranda/ui/ProjectDialog.java [hunk 2]
```diff
@@ -350,9 +368,21 @@ void edButton_actionPerformed(ActionEvent e) {
...
+    	Dimension psdlgSize = new Dimension(550,650);
+    	psdlg.setMinimumSize(psdlgSize);
+    	Dimension frmSize = App.getFrame().getSize();
+        Point loc = App.getFrame().getLocation();
+        psdlg.setLocation((frmSize.width - psdlgSize.width) / 2 + loc.x, (frmSize.height - psdlgSize.height) / 2 + loc.y);
+    	psdlg.setVisible(true);
+    	newSummary = ((PlanSummaryDialog) psdlg).getData();
+    	psdlg.dispose();
+    }
+
```
Comment (krcarey1) [inline src/net/sf/memoranda/ui/ProjectDialog.java:383]: formatting within method

**Comment by smbende1** [comment]
Final formatting changes. I've noticed that changes made in a text editor just mess up the formatting in Eclipse so I'm no longer going to try keeping the formatting looking proper in both places on the assumption that any formal review will happen in Eclipse anyway.

**Comment by dtapia4** [comment]
Looks good.

**Pull request merged**